### PR TITLE
[improve] [pip] PIP-357: Correct the conf name in load balance module.

### PR DESCRIPTION
**Pull request by thetumbled**
PIP: #22823

### Motivation

We use `loadBalancerBandwithInResourceWeight` and `loadBalancerBandwithOutResourceWeight` to calculate the broker's load in the load balance module. However, the correct conf name should be `loadBalancerBandwidthInResourceWeight` and `loadBalancerBandwidthOutResourceWeight`. This PIP is to correct the conf name in the load balance module.

### Modifications

Rename `loadBalancerBandwithInResourceWeight` to `loadBalancerBandwidthInResourceWeight` and `loadBalancerBandwithOutResourceWeight` to `loadBalancerBandwidthOutResourceWeight` in the load balance module.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

*If the box was checked, please highlight the changes*

- [ ] Dependencies (add or upgrade a dependency)
- [ ] The public API
- [ ] The schema
- [ ] The default values of configurations
- [ ] The threading model
- [ ] The binary protocol
- [ ] The REST endpoints
- [ ] The admin CLI options
- [ ] The metrics
- [ ] Anything that affects deployment

### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete` 

### Matching PR in forked repository

PR in forked repository: https://github.com/thetumbled/pulsar/pull/56

### CODE REVIEW

File: pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java [hunk 1]
```diff
@@ -2444,17 +2444,60 @@ The max allowed delay for delayed delivery (in milliseconds). If the broker rece
...
+     * @return
+     */
+    public double getLoadBalancerBandwidthInResourceWeight() {
+        if (loadBalancerBandwidthInResourceWeight != 1.0) {
+            return loadBalancerBandwidthInResourceWeight;
+        } else if (loadBalancerBandwithInResourceWeight != 1.0) {
+            return loadBalancerBandwithInResourceWeight;
+        } else {
+            return 1.0;
+        }
```
Comment (Demogorgon314) [inline pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java:2488]:
```suggestion
        if (loadBalancerBandwidthInResourceWeight != 1.0) {
            return loadBalancerBandwidthInResourceWeight;
        }
        if (loadBalancerBandwithInResourceWeight != 1.0) {
            return loadBalancerBandwithInResourceWeight;
        }
        return 1.0;
```

File: pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java [hunk 2]
```diff
@@ -2444,17 +2444,60 @@ The max allowed delay for delayed delivery (in milliseconds). If the broker rece
...
+    }
+
+    public double getLoadBalancerBandwidthOutResourceWeight() {
+        if (loadBalancerBandwidthOutResourceWeight != 1.0) {
```
Comment (Demogorgon314) [inline pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java:2492]: Same as the previous comment.

Comment (thetumbled) [reply pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java:2492]: Done.

**Review by Demogorgon314 — APPROVED** [review]

**Review by Technoboy- — APPROVED** [review]

**Review by BewareMyPower — APPROVED** [review]

**Pull request merged**